schema.graphql

```diff
@@ -15,6 +15,7 @@ type Hat @entity {
   wearers: [Wearer!]!
   admin: Hat!
   badStandings: [Wearer!]!
+  requestedLinkFromTree: Tree @derivedFrom(field: "requestedLinkToAdminHat")
```

Comment:
This name is a touch ambiguous; "requested" makes it sound like this entity is the one that did the requesting. So I'd suggest changing to more of a noun, like "linkRequestFromTree".